tests/js/client/shell/shell-api-users.js

```diff
@@ -0,0 +1,793 @@
+
+/* jshint globalstrict:false, strict:false, maxlen: 200 */
+/* global db, fail, arango, assertTrue, assertFalse, assertEqual, assertNotUndefined */
+
+// //////////////////////////////////////////////////////////////////////////////
+// / @brief 
+// /
+// /
+// / DISCLAIMER
+// /
+// / Copyright 2018 ArangoDB GmbH, Cologne, Germany
+// /
+// / Licensed under the Apache License, Version 2.0 (the "License")
+// / you may not use this file except in compliance with the License.
+// / You may obtain a copy of the License at
+// /
+// /     http://www.apache.org/licenses/LICENSE-2.0
+// /
+// / Unless required by applicable law or agreed to in writing, software
+// / distributed under the License is distributed on an "AS IS" BASIS,
+// / WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// / See the License for the specific language governing permissions and
+// / limitations under the License.
+// /
+// / Copyright holder is ArangoDB GmbH, Cologne, Germany
+// /
+// / @author 
+// //////////////////////////////////////////////////////////////////////////////
+
+'use strict';
+
+const internal = require('internal');
+const sleep = internal.sleep;
+const forceJson = internal.options().hasOwnProperty('server.force-json') && internal.options()['server.force-json'];
+const contentType = forceJson ? "application/json" :  "application/x-velocypack";
+const jsunity = require("jsunity");
+
+let api = "/_api/user";
+
+function user_managementSuite () {
+  return {
+    setUp: function() {
+      for (let i = 1; i < 10; i++) {
+        arango.DELETE_RAW("/_api/user/users-" + i);;
+      }
+    },
+
+    tearDown: function() {
+      for (let i = 1; i < 10; i++) {
+        arango.DELETE_RAW("/_api/user/users-" + i.to_s);;
```

Comment:
`to_s` does not exist in JavaScript
```suggestion
        arango.DELETE_RAW("/_api/user/users-" + i);
```